packages/fxa-content-server/app/scripts/lib/experiments/grouping-rules/communication-prefs.js

```diff
@@ -10,22 +10,24 @@
 
 const BaseGroupingRule = require('./base');
 
-const AVAILABLE_LANGUAGES = [
-  'de',
-  'en',
-  'en-[a-z]{2}',
-  'es',
-  'es-[a-z]{2}',
-  'fr',
-  'hu',
-  'id',
-  'pl',
-  'pt-br',
```

Comment:
This one may be problematic because Brazillian Portuguese is significantly different to Portuguese from Portugal.

Reply:
Yup, note that this is the deleted line, not the new line.